### PR TITLE
fix: #998, #992 Setting whole app to work only in portrait mode.

### LINKED ISSUE #998: Setting app to work only in Portrait Mode
**Summary:**

The app has many bugs when working in landscape mode and also on device rotation. It will be better if we restrict all activities to work in portrait mode only.
This would solve many small bugs throughout the app.

### PR DESCRIPTION
**Pull request by rahul-jha98**
Fixes #998 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by luckyman20** [comment]
@rahul-jha98 Checkout this [PR](https://github.com/openMF/mobile-wallet/pull/168) from `mobile-wallet` repository and update your PR.

**Comment by luckyman20** [comment]
@rahul-jha98 Squash your commits.

**Comment by rahul-jha98** [comment]
I am getting thin error in Travis Build. 
screenshot from 2019-02-09 20-45-59

@luckyman20 Can you guide me on how should I resolve this?

The solution I searched online were for the local build checks.

**Comment by luckyman20** [comment]
Run `./gradlew build` (if you are on Linux or MacOS) or `gradlew build` (if you are on Windows), then try to resolve this locally and then push the changes.

**Comment by RajaVamsi11** [comment]
@rahul-jha98 Are you still working on fixing the errors in Travis Build? or I am interested in working on this issue.

**Comment by RajaVamsi11** [comment]
@luckyman20 @rahul-jha98 I think that this issue must be solved asap because solving this issue will indirectly solve other issues like #992 #1018 #1024.

**Comment by rahul-jha98** [comment]
@luckyman20 Thanks you for guiding me. I would have done it sooner but got busy in college course work. Sorry for being late.

File: app/src/main/java/org/mifos/mobilebanking/ui/activities/base/BaseActivity.java [hunk 1]
```diff
@@ -40,6 +41,8 @@
     @Override
     public void setContentView(int layoutResID) {
         super.setContentView(layoutResID);
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
```
Comment (luckyman20) [inline app/src/main/java/org/mifos/mobilebanking/ui/activities/base/BaseActivity.java]:
```suggestion
```

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -30,7 +30,7 @@
             android:name="org.mifos.mobilebanking.ui.activities.LoginActivity"
             android:configChanges="orientation|screenSize"
             android:windowSoftInputMode="adjustResize"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name" />
```
Comment (luckyman20) [inline app/src/main/AndroidManifest.xml:33]: Remove these kind of changes.

**Comment by shiv07tiwari** [comment]
@rahul-jha98 Are you still working on this ?

**Comment by rahul-jha98** [comment]
No, you can work on the issue.

**Comment by shiv07tiwari** [comment]
@rahul-jha98 Okay I'll fix this one

**Pull request closed without merging**